gulpfile.js

```diff
@@ -165,10 +166,17 @@ gulp.task('html', function() {
 // for image compression
 gulp.task('images', function() {
     return gulp.src('frontend/src/images/**/*')
-        .pipe(gulp_if(flags.production, imagemin({ optimizationLevel: 3, progressive: true, interlaced: true })))
+        .pipe(gulp_if(flags.production, imagemin({ optimizationLevel: 5, progressive: true, interlaced: true })))
         .pipe(gulp.dest('frontend/dist/images'));
 });
 
+// for image of organization folder
+gulp.task('imagesorgan', function() {
```

Comment:
No need to write this task, as ** will take nested folders too in `images` task.
Please remove this task. @gautamjajoo